spark/src/main/scala/io/ddf/spark/etl/TransformationHandler.scala

```diff
@@ -322,10 +323,17 @@ class TransformationHandler(mDDF: DDF) extends CoreTransformationHandler(mDDF) {
 
   override def oneHotEncoding(inputColumn: String, outputColumnName: String): DDF = {
     val df = this.mDDF.getRepresentationHandler.get(classOf[DataFrame]).asInstanceOf[DataFrame]
+    val newCol = this.mDDF.getColumn(inputColumn) match {
+      case x if(x.getType == Schema.ColumnType.DOUBLE) => df.col(inputColumn)
+      case x if(x.getType != Schema.ColumnType.VECTOR && x.isNumeric)
```

Comment:
I am just curious about that VECTOR type, @Huandao0812. What does the value look like?